docs/resources/virtualmachine.md

```diff
@@ -64,12 +64,12 @@ resource "harvester_virtualmachine" "ubuntu20-dev" {
   cpu    = 2
   memory = "2Gi"
 
-  start        = true
+  run_strategy = "Always"
```

Comment:
the default run strategy in Harvester is `RerunOnFailure`

Reply:
If use RerunOnFailure，user can't start/stop vm by terraform

Reply:
both `Always` and `RerunOnFailure` will have the same behavior of start and stop, are we using sub-resource action to manage it?
<img width="755" alt="image" src="https://user-images.githubusercontent.com/4569037/167236077-7038c0a3-6a1a-4582-8666-f104435d5ad7.png">

Reply:
> are we using sub-resource action to manage it?
No, RerunOnFailure  is ok. I thought RerunOnFailure's behavior is like Manual before.

Changed to RerunOnFailure